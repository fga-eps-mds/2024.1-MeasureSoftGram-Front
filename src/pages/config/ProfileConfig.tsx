import useRequireAuth from '@hooks/useRequireAuth';
import { NextPageWithLayout } from '@pages/_app.next';
import Head from 'next/head';
import { getLayout } from '@components/Layout/getLayout';
import { Container, TextField, InputAdornment, IconButton } from '@mui/material';
import { Visibility, VisibilityOff, ContentCopy } from '@mui/icons-material';
import { useEffect, useState } from 'react';
import { getAccessToken } from '@services/Auth';
import { useTranslation } from 'react-i18next';

const ProfileConfig: NextPageWithLayout = () => {
  useRequireAuth();

  const [tokenVisible, setTokenVisible] = useState(false);
  const [apiToken, setApiToken] = useState('');

  useEffect(() => {
    getAccessToken().then((res) => {
      if (res.type === 'success') {
        setApiToken(`Token ${res.value.key}`); // Changed to use template literals
      }
    });
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);

  const handleToggleVisibility = () => {
    setTokenVisible(!tokenVisible);
  };

  const handleCopyToken = () => {
    navigator.clipboard.writeText(apiToken);
  };

  const { t } = useTranslation('settings');

  return (
    <>
      <Head>
        <title>Config</title>
      </Head>
      <Container>
        <h1>{t('pageTitle')}</h1>

        <h3 style={{ textAlign: 'left' }}>{t('apiKeyHeader')}</h3>
        <TextField
          style={{ textAlign: 'center' }}
          type={tokenVisible ? 'text' : 'password'}
          value={apiToken}
          label={t('apiKeyLabel')}
          InputProps={{
            endAdornment: (
              <InputAdornment position="end">
                <IconButton aria-label="toggle password visibility" onClick={handleToggleVisibility}>
                  {tokenVisible ? <VisibilityOff /> : <Visibility />}
                </IconButton>
                <IconButton aria-label="copy api token" onClick={handleCopyToken}>
                  <ContentCopy />
                </IconButton>
              </InputAdornment>
            )
          }}
        />
      </Container>
    </>
  );
};

ProfileConfig.getLayout = getLayout;

export default ProfileConfig;
