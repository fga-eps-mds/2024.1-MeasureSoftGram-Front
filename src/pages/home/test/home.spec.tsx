import React from 'react';
import { render } from '@testing-library/react';
import { AuthProvider } from '@contexts/Auth';
import { useTranslation } from 'react-i18next';
import { ni18nConfig } from 'n18n.config';
import Home from '../index.page';


describe('Home', () => {
  describe('Snapshot', () => {
    it('Deve corresponder ao Snapshot', () => {
      // Mock the react-i18next module
      jest.mock('react-i18next', () => ({
        useTranslation: () => ({
          t: (key) => key, // Simply return the key instead of translating
          i18n: {
            changeLanguage: jest.fn(), // Mock the changeLanguage function
          },
        }),
      }));

      const tree = render(<AuthProvider>
        <Home />
      </AuthProvider>);

      expect(tree).toMatchSnapshot();
    });
  });
});
