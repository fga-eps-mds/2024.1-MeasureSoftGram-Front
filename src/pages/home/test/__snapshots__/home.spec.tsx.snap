// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snapshot Deve corresponder ao Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="MuiBox-root css-1xhj18k"
        >
          <div
            class="MuiBox-root css-bb2sw8"
          >
            <div
              class="MuiBox-root css-k008qs"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
                style="color: rgb(51, 86, 142); font-weight: bold;"
              >
                homepage.sub-title
              </h4>
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                style="font-size: 16px;"
              >
                homepage.description
              </p>
            </div>
            <div
              class="MuiBox-root css-1n8etdr"
            >
              <nav
                class="MuiList-root MuiList-padding css-h4y409-MuiList-root"
              >
                <div
                  class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters Mui-selected MuiListItemButton-root MuiListItemButton-gutters Mui-selected css-llb1ir-MuiButtonBase-root-MuiListItemButton-root"
                  role="button"
                  tabindex="0"
                >
                  <div
                    class="MuiBox-root css-sjfhw0"
                  >
                    <div
                      class="MuiBox-root css-1g9x5we"
                    >
                      <div
                        class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                          data-testid="EastOutlinedIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="m15 5-1.41 1.41L18.17 11H2v2h16.17l-4.59 4.59L15 19l7-7-7-7z"
                          />
                        </svg>
                      </div>
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body1 button-text css-1ixorw4-MuiTypography-root"
                    >
                      homepage.organization.id
                    </p>
                  </div>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </div>
                <div
                  class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-yzx94z-MuiButtonBase-root-MuiListItemButton-root"
                  role="button"
                  tabindex="0"
                >
                  <div
                    class="MuiBox-root css-sjfhw0"
                  >
                    <div
                      class="MuiBox-root css-1g9x5we"
                    />
                    <p
                      class="MuiTypography-root MuiTypography-body1 button-text css-1ixorw4-MuiTypography-root"
                    >
                      homepage.repository.id
                    </p>
                  </div>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </div>
                <div
                  class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-yzx94z-MuiButtonBase-root-MuiListItemButton-root"
                  role="button"
                  tabindex="0"
                >
                  <div
                    class="MuiBox-root css-sjfhw0"
                  >
                    <div
                      class="MuiBox-root css-1g9x5we"
                    />
                    <p
                      class="MuiTypography-root MuiTypography-body1 button-text css-1ixorw4-MuiTypography-root"
                    >
                      homepage.pre-config.id
                    </p>
                  </div>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </div>
                <div
                  class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-yzx94z-MuiButtonBase-root-MuiListItemButton-root"
                  role="button"
                  tabindex="0"
                >
                  <div
                    class="MuiBox-root css-sjfhw0"
                  >
                    <div
                      class="MuiBox-root css-1g9x5we"
                    />
                    <p
                      class="MuiTypography-root MuiTypography-body1 button-text css-1ixorw4-MuiTypography-root"
                    >
                      homepage.config.id
                    </p>
                  </div>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </div>
              </nav>
            </div>
          </div>
          <div
            class="MuiBox-root css-euotp0"
          >
            <div
              class="MuiBox-root css-1mwjrl3"
            >
              <div
                class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18vk9q-MuiPaper-root-MuiCard-root"
                id="homepage.organization.id"
              >
                <div
                  class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
                >
                  <div
                    class="MuiBox-root css-1glkofm"
                  >
                    <div
                      class="MuiBox-root css-pw8lt9"
                      id="images"
                    >
                      <img
                        alt="green iguana"
                        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                        src="/images/png/structure.png"
                      />
                    </div>
                    <div
                      class="MuiBox-root css-s4ej5b"
                      id="texts"
                    >
                      <div
                        class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                      >
                        homepage.organization.title
                      </div>
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                      >
                        homepage.organization.description
                      </p>
                      <div
                        class="MuiBox-root css-1vp8jwk"
                      >
                        <div
                          class="MuiBox-root css-0"
                        >
                          <div
                            class="MuiBox-root css-pb7e80"
                          >
                            <button
                              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
                              tabindex="0"
                              type="button"
                            >
                              homepage.organization.view-organization
                              <span
                                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                          <div
                            class="MuiBox-root css-t3r43i"
                          >
                            <button
                              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
                              tabindex="0"
                              type="button"
                            >
                              homepage.organization.create-organization
                              <span
                                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
                >
                  <div
                    class="MuiBox-root css-1glkofm"
                  >
                    <div
                      class="MuiBox-root css-pw8lt9"
                      id="images"
                    >
                      <img
                        alt="green iguana"
                        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                        src="/images/png/development.png"
                      />
                    </div>
                    <div
                      class="MuiBox-root css-s4ej5b"
                      id="texts"
                    >
                      <div
                        class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                      >
                        homepage.product.title
                      </div>
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                      >
                        homepage.product.description
                      </p>
                      <div
                        class="MuiBox-root css-1vp8jwk"
                      >
                        <div
                          class="MuiBox-root css-pb7e80"
                        >
                          <button
                            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
                            tabindex="0"
                            type="button"
                          >
                            homepage.product.create-product
                            <span
                              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                            />
                          </button>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18vk9q-MuiPaper-root-MuiCard-root"
                id="homepage.repository.id"
              >
                <div
                  class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
                >
                  <div
                    class="MuiBox-root css-1glkofm"
                  >
                    <div
                      class="MuiBox-root css-pw8lt9"
                      id="images"
                    >
                      <img
                        alt="green iguana"
                        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                        src="/images/png/folders.png"
                      />
                    </div>
                    <div
                      class="MuiBox-root css-s4ej5b"
                      id="texts"
                    >
                      <div
                        class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                      >
                        homepage.repository.title
                      </div>
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                      >
                        homepage.repository.description
                      </p>
                      <div
                        class="MuiBox-root css-1vp8jwk"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
                >
                  <div
                    class="MuiBox-root css-1glkofm"
                  >
                    <div
                      class="MuiBox-root css-pw8lt9"
                      id="images"
                    >
                      <img
                        alt="green iguana"
                        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                        src="/images/png/new-offer.png"
                      />
                    </div>
                    <div
                      class="MuiBox-root css-s4ej5b"
                      id="texts"
                    >
                      <div
                        class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                      >
                        homepage.release.title
                      </div>
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                      >
                        homepage.release.description
                      </p>
                      <div
                        class="MuiBox-root css-1vp8jwk"
                      />
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18vk9q-MuiPaper-root-MuiCard-root"
                id="homepage.pre-config.id"
              >
                <div
                  class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
                >
                  <div
                    class="MuiBox-root css-1glkofm"
                  >
                    <div
                      class="MuiBox-root css-pw8lt9"
                      id="images"
                    >
                      <img
                        alt="green iguana"
                        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                        src="/images/png/setting.png"
                      />
                    </div>
                    <div
                      class="MuiBox-root css-s4ej5b"
                      id="texts"
                    >
                      <div
                        class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                      >
                        homepage.pre-config.title
                      </div>
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                      >
                        homepage.pre-config.description
                      </p>
                      <div
                        class="MuiBox-root css-1vp8jwk"
                      />
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18vk9q-MuiPaper-root-MuiCard-root"
                id="homepage.config.id"
              >
                <div
                  class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
                >
                  <div
                    class="MuiBox-root css-1glkofm"
                  >
                    <div
                      class="MuiBox-root css-pw8lt9"
                      id="images"
                    >
                      <img
                        alt="green iguana"
                        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                        src="/images/png/web-management.png"
                      />
                    </div>
                    <div
                      class="MuiBox-root css-s4ej5b"
                      id="texts"
                    >
                      <div
                        class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                      >
                        homepage.config.title
                      </div>
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                      >
                        homepage.config.description
                      </p>
                      <div
                        class="MuiBox-root css-1vp8jwk"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-1xhj18k"
      >
        <div
          class="MuiBox-root css-bb2sw8"
        >
          <div
            class="MuiBox-root css-k008qs"
          >
            <h4
              class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
              style="color: rgb(51, 86, 142); font-weight: bold;"
            >
              homepage.sub-title
            </h4>
          </div>
          <div
            class="MuiBox-root css-0"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              style="font-size: 16px;"
            >
              homepage.description
            </p>
          </div>
          <div
            class="MuiBox-root css-1n8etdr"
          >
            <nav
              class="MuiList-root MuiList-padding css-h4y409-MuiList-root"
            >
              <div
                class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters Mui-selected MuiListItemButton-root MuiListItemButton-gutters Mui-selected css-llb1ir-MuiButtonBase-root-MuiListItemButton-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiBox-root css-sjfhw0"
                >
                  <div
                    class="MuiBox-root css-1g9x5we"
                  >
                    <div
                      class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="EastOutlinedIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="m15 5-1.41 1.41L18.17 11H2v2h16.17l-4.59 4.59L15 19l7-7-7-7z"
                        />
                      </svg>
                    </div>
                  </div>
                  <p
                    class="MuiTypography-root MuiTypography-body1 button-text css-1ixorw4-MuiTypography-root"
                  >
                    homepage.organization.id
                  </p>
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <div
                class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-yzx94z-MuiButtonBase-root-MuiListItemButton-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiBox-root css-sjfhw0"
                >
                  <div
                    class="MuiBox-root css-1g9x5we"
                  />
                  <p
                    class="MuiTypography-root MuiTypography-body1 button-text css-1ixorw4-MuiTypography-root"
                  >
                    homepage.repository.id
                  </p>
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <div
                class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-yzx94z-MuiButtonBase-root-MuiListItemButton-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiBox-root css-sjfhw0"
                >
                  <div
                    class="MuiBox-root css-1g9x5we"
                  />
                  <p
                    class="MuiTypography-root MuiTypography-body1 button-text css-1ixorw4-MuiTypography-root"
                  >
                    homepage.pre-config.id
                  </p>
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <div
                class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-yzx94z-MuiButtonBase-root-MuiListItemButton-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiBox-root css-sjfhw0"
                >
                  <div
                    class="MuiBox-root css-1g9x5we"
                  />
                  <p
                    class="MuiTypography-root MuiTypography-body1 button-text css-1ixorw4-MuiTypography-root"
                  >
                    homepage.config.id
                  </p>
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
            </nav>
          </div>
        </div>
        <div
          class="MuiBox-root css-euotp0"
        >
          <div
            class="MuiBox-root css-1mwjrl3"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18vk9q-MuiPaper-root-MuiCard-root"
              id="homepage.organization.id"
            >
              <div
                class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
              >
                <div
                  class="MuiBox-root css-1glkofm"
                >
                  <div
                    class="MuiBox-root css-pw8lt9"
                    id="images"
                  >
                    <img
                      alt="green iguana"
                      class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                      src="/images/png/structure.png"
                    />
                  </div>
                  <div
                    class="MuiBox-root css-s4ej5b"
                    id="texts"
                  >
                    <div
                      class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                    >
                      homepage.organization.title
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                    >
                      homepage.organization.description
                    </p>
                    <div
                      class="MuiBox-root css-1vp8jwk"
                    >
                      <div
                        class="MuiBox-root css-0"
                      >
                        <div
                          class="MuiBox-root css-pb7e80"
                        >
                          <button
                            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
                            tabindex="0"
                            type="button"
                          >
                            homepage.organization.view-organization
                            <span
                              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                            />
                          </button>
                        </div>
                        <div
                          class="MuiBox-root css-t3r43i"
                        >
                          <button
                            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
                            tabindex="0"
                            type="button"
                          >
                            homepage.organization.create-organization
                            <span
                              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                            />
                          </button>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
              >
                <div
                  class="MuiBox-root css-1glkofm"
                >
                  <div
                    class="MuiBox-root css-pw8lt9"
                    id="images"
                  >
                    <img
                      alt="green iguana"
                      class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                      src="/images/png/development.png"
                    />
                  </div>
                  <div
                    class="MuiBox-root css-s4ej5b"
                    id="texts"
                  >
                    <div
                      class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                    >
                      homepage.product.title
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                    >
                      homepage.product.description
                    </p>
                    <div
                      class="MuiBox-root css-1vp8jwk"
                    >
                      <div
                        class="MuiBox-root css-pb7e80"
                      >
                        <button
                          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
                          tabindex="0"
                          type="button"
                        >
                          homepage.product.create-product
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18vk9q-MuiPaper-root-MuiCard-root"
              id="homepage.repository.id"
            >
              <div
                class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
              >
                <div
                  class="MuiBox-root css-1glkofm"
                >
                  <div
                    class="MuiBox-root css-pw8lt9"
                    id="images"
                  >
                    <img
                      alt="green iguana"
                      class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                      src="/images/png/folders.png"
                    />
                  </div>
                  <div
                    class="MuiBox-root css-s4ej5b"
                    id="texts"
                  >
                    <div
                      class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                    >
                      homepage.repository.title
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                    >
                      homepage.repository.description
                    </p>
                    <div
                      class="MuiBox-root css-1vp8jwk"
                    />
                  </div>
                </div>
              </div>
              <div
                class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
              >
                <div
                  class="MuiBox-root css-1glkofm"
                >
                  <div
                    class="MuiBox-root css-pw8lt9"
                    id="images"
                  >
                    <img
                      alt="green iguana"
                      class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                      src="/images/png/new-offer.png"
                    />
                  </div>
                  <div
                    class="MuiBox-root css-s4ej5b"
                    id="texts"
                  >
                    <div
                      class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                    >
                      homepage.release.title
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                    >
                      homepage.release.description
                    </p>
                    <div
                      class="MuiBox-root css-1vp8jwk"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18vk9q-MuiPaper-root-MuiCard-root"
              id="homepage.pre-config.id"
            >
              <div
                class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
              >
                <div
                  class="MuiBox-root css-1glkofm"
                >
                  <div
                    class="MuiBox-root css-pw8lt9"
                    id="images"
                  >
                    <img
                      alt="green iguana"
                      class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                      src="/images/png/setting.png"
                    />
                  </div>
                  <div
                    class="MuiBox-root css-s4ej5b"
                    id="texts"
                  >
                    <div
                      class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                    >
                      homepage.pre-config.title
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                    >
                      homepage.pre-config.description
                    </p>
                    <div
                      class="MuiBox-root css-1vp8jwk"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18vk9q-MuiPaper-root-MuiCard-root"
              id="homepage.config.id"
            >
              <div
                class="MuiCardContent-root css-1hpgiqs-MuiCardContent-root"
              >
                <div
                  class="MuiBox-root css-1glkofm"
                >
                  <div
                    class="MuiBox-root css-pw8lt9"
                    id="images"
                  >
                    <img
                      alt="green iguana"
                      class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-14x8w58-MuiCardMedia-root"
                      src="/images/png/web-management.png"
                    />
                  </div>
                  <div
                    class="MuiBox-root css-s4ej5b"
                    id="texts"
                  >
                    <div
                      class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter MuiTypography-gutterBottom css-1g00ex-MuiTypography-root"
                    >
                      homepage.config.title
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
                    >
                      homepage.config.description
                    </p>
                    <div
                      class="MuiBox-root css-1vp8jwk"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
